src/components/single-page/single-page-content.tsx

```diff
@@ -17,39 +15,22 @@ interface IProps {
 
 export const SinglePageContent: React.FC<IProps> = (props) => {
   const { activity, teacherEditionMode, pluginsLoaded } = props;
-  let questionNumber = 0;
-  let embeddableNumber = 0;
-
+  const questionNumber = 0;
```

Comment:
`questionNumber` is always 0, and then we add it to our embeddableQuestionNumberStart.

It seems like that was something carried over that we can get rid of that completely.